lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -130,20 +131,26 @@ class TurboGraft.Remote
 
     unless TurboGraft.hasTGAttribute(@initiator, 'tg-remote-norefresh')
       if @opts.fullRefresh && @refreshOnSuccess
-        Page.refresh(onlyKeys: @refreshOnSuccess)
+        Page.refresh
+          onlyKeys: @refreshOnSuccess
+          updatePushState: @updatePushState
```

Comment:
I don't like manually passing these in every call. I had to do it in #162 as well, so I went with:
![image](https://cloud.githubusercontent.com/assets/4889856/18962128/89d5def2-863d-11e6-9d35-99fd8782d0e8.png)

It might be even cleaner to just make a function that wraps `Page.refresh` and always forwards the arguments we want instead of calling it using the result of another function though.